lib/auth/tls_test.go

```diff
@@ -3007,6 +3007,37 @@ func TestClusterAlertAccessControls(t *testing.T) {
 
 	_, err = nopClt.GetClusterAlerts(ctx, types.GetClusterAlertsRequest{})
 	require.True(t, trace.IsAccessDenied(err))
+
+	// add more alerts that use resource verb permits
+	alert3, err := types.NewClusterAlert(
+		"alert-3",
+		"msg 3",
+		types.WithAlertLabel(types.AlertVerbPermit, "token:create"),
+	)
+	require.NoError(t, err)
+
+	alert4, err := types.NewClusterAlert(
+		"alert-4",
+		"msg 4",
+		types.WithAlertLabel(types.AlertVerbPermit, "token:create|role:read"),
+	)
+	require.NoError(t, err)
+
+	err = adminClt.UpsertClusterAlert(ctx, alert3)
+	require.NoError(t, err)
+
+	err = adminClt.UpsertClusterAlert(ctx, alert4)
+	require.NoError(t, err)
+
+	// verify that admin client can see all alerts
+	alerts, err = adminClt.GetClusterAlerts(ctx, types.GetClusterAlertsRequest{})
+	require.NoError(t, err)
+	require.Len(t, alerts, 4)
+
+	// verify that node client can only see one of the two new alerts
```

Comment:
Can we add an assertion to check that it sees the correct alerts and not just the correct number of alerts?